orders/tests.py

```diff
@@ -0,0 +1,79 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+from django.core.exceptions import ObjectDoesNotExist
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(pk=1),
+                        daycare_id=DayCare.objects.get(pk=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        total_price=TOTAL_PRICE,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self, order):
+        new_order = Order.create(dog_owner_id=order.dog_owner_id,
+                                 daycare_id=order.daycare_id,
+                                 starting_date=order.starting_date,
+                                 finishing_date=order.finishing_date,
+                                 total_price=order.total_price)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        assert order in Order.objects.all()
+        order.delete()
+        assert order not in Order.objects.all()
+
+    def test_get_all_orders_by_dog_owner_id(self, order):
+        dog_owner_orders = Order.get_all_dog_owner_orders(order.dog_owner_id)
+
+        for dog_owner_order in dog_owner_orders:
+            assert dog_owner_order in Order.objects.all()
+
+    def test_get_all_orders_by_daycare_id(self, order):
+        daycare_orders = Order.get_all_daycare_orders(order.daycare_id)
+
+        for daycare_order in daycare_orders:
+            assert daycare_order in Order.objects.all()
+
+    def test_get_order_by_order_id(self, order):
+        order = Order.objects.filter(order_id=order.order_id).first()
+        assert order is not None
+
+    def test_get_not_existed_order_by_order_id(self):
+        try:
+            Order.objects.get(order_id=-1)
```

Comment:
order_id 1 is created in the data migration no? if not the test will be effected by the data migration
Maybe create new order and than you will know the last id that was created

Reply:
> order_id 1 is created in the data migration no? if not the test will be effected by the data migration Maybe create new order and than you will know the last id that was created

The test is asking for order_id = -1, which is not possible. That's why the "get" method will throw an exception, and the test will pass.